lib/mix/tasks/firmware.burn.ex

```diff
@@ -42,6 +42,7 @@ defmodule Mix.Tasks.Firmware.Burn do
     config = Mix.Project.config
     otp_app = config[:app]
     target = config[:target]
+    images_path = config[:images_path] || config[:build_path] || "_build/images"
```

Comment:
actually we would need to expand it if it is passed.

images_path = 
  (config[:images_path] || Path.join([Mix.Project.build_path, "nerves", "images"]))
  |> Path.expand